### PR TITLE
[WIP] Code refactoring using StyleCop.

### PR DESCRIPTION
**Pull request by kegor**
Replicated changes from https://github.com/groupdocs-viewer/GroupDocs.Viewer-for-.NET-MVC/pull/64.

### CODE REVIEW

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/Products/Viewer/Cache/MemoryPageStreamFactory.cs  1
- src/Products/Viewer/Controllers/ViewerApiController.cs  3
         

Clones removed
==============
+ src/Products/Viewer/Config/ViewerConfiguration.cs  -2
+ src/Products/Viewer/Controllers/ViewerApiController.cs  -3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5914669&bid=19379271)

File: src/Products/Viewer/Controllers/ViewerApiController.cs [hunk 1]
```diff
@@ -1,47 +1,52 @@
...
+        private static Common.Config.GlobalConfiguration globalConfiguration;
 
         /// <summary>
-        /// Constructor
+        /// Initializes a new instance of the <see cref="ViewerApiController"/> class.
         /// </summary>
         public ViewerApiController()
         {
+            // Check if filesDirectory is relative or absolute path
+            globalConfiguration = new Common.Config.GlobalConfiguration();
```
Comment (bobkovalex) [inline src/Products/Viewer/Controllers/ViewerApiController.cs:48]: Codacy found an issue: [Remove this assignment of 'globalConfiguration' or initialize it statically.](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5914669)

Comment (egorovpavel) [inline src/Products/Viewer/Controllers/ViewerApiController.cs:48]: Codacy found an issue: [Remove this assignment of 'globalConfiguration' or initialize it statically.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5914668)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/Products/Viewer/Cache/MemoryPageStreamFactory.cs  1
- src/Products/Viewer/Controllers/ViewerApiController.cs  3
         

Clones removed
==============
+ src/Products/Viewer/Config/ViewerConfiguration.cs  -2
+ src/Products/Viewer/Controllers/ViewerApiController.cs  -3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5914668&bid=19379270)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/Products/Viewer/Controllers/ViewerApiController.cs  3
- src/Products/Viewer/Cache/MemoryPageStreamFactory.cs  1
         

Clones removed
==============
+ src/Products/Viewer/Config/ViewerConfiguration.cs  -2
+ src/Products/Viewer/Controllers/ViewerApiController.cs  -3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5914669&bid=19379271)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/Products/Viewer/Cache/MemoryPageStreamFactory.cs  1
- src/Products/Viewer/Controllers/ViewerApiController.cs  3
         

Clones removed
==============
+ src/Products/Viewer/Controllers/ViewerApiController.cs  -3
+ src/Products/Viewer/Config/ViewerConfiguration.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5914668&bid=19379270)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/Products/Viewer/Controllers/ViewerApiController.cs  3
- src/Products/Viewer/Cache/MemoryPageStreamFactory.cs  1
         

Clones removed
==============
+ src/Products/Viewer/Controllers/ViewerApiController.cs  -5
+ src/Products/Viewer/Config/ViewerConfiguration.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5914668&bid=19379270)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/Products/Viewer/Controllers/ViewerApiController.cs  3
- src/Products/Viewer/Cache/MemoryPageStreamFactory.cs  1
         

Clones removed
==============
+ src/Products/Viewer/Controllers/ViewerApiController.cs  -5
+ src/Products/Viewer/Config/ViewerConfiguration.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5914669&bid=19379271)

**Pull request merged**